src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -26,6 +27,7 @@
     implements Comparable<CommentAttributes> {
 
     private Long commentId = null;
+    private SimpleDateFormat sdf = new SimpleDateFormat("EEE, dd MMM YYYY, HH:mm zzz");
```

Comment:
1. You could add the seconds also if you wish.
2. In fact, move this to `Const.SystemParams`. I'm going to feed from it for my #3852 (or another PR since changing the display format in #3852 right now might make it too big) once it's merged.